scripts/parse-word-break.js

```diff
@@ -1,43 +1,39 @@
 'use strict';
 
 const utils = require('./utils.js');
-
-const handled = new Set();
```

Comment:
The `handled` set are shared between different Unicode versions, so previously we generated incorrect `Word_Break/Other` data because there is certain character, e.g. U+FE10 is in WordBreakProperty.txt as of Unicode 15.1.0 but removed as of Unicode 16.0.

Reply:
Woah, good catch!